<launch>
    <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy" output="screen">    
  	<param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

	<node name="PlannerNode" pkg="rl_nav" type="PlannerNode.py" output="screen" respawn="true"/>

	<node name="rl_nav" pkg="rl_nav" type="rl_nav" output="screen" respawn="true">
		<param name="predictor" type="string" value="RL" />
		<param name="qThresh" type="double" value="-0.2" />	
		<param name="vel_scale" type="double" value="1.0" />	
		<param name="mode" type="string" value="TRAIN" />
		<param name="num_episodes" type="int" value="10" />
		<param name="num_actions" type="int" value="14" />
		<param name="max_steps" type="int" value="1000" />
		<param name="map" type="int" value="0" />
		<param name="init_x" type="double" value="-1" />
		<param name="init_y" type="double" value="1" />
		<param name="init_Y" type="double" value="0.4" />
	</node>

	
	
</launch>
